package.json

```diff
@@ -22,6 +22,7 @@
     "nuxt": "^3.13.2",
     "nuxt-gtag": "^3.0.1",
     "postprocessing": "^6.36.3",
+    "qss": "^3.0.0",
```

Comment:
Please add the requirement to install this library in the documentation of link-preview.

Refer to https://inspira-ui.com/components/particle-whirlpool-bg on how extra dependencies are documented.